src/components/ManageUsers/manage-users.css

```diff
@@ -0,0 +1,84 @@
+.usersearch__h1 {
+  font-size: 16px;
+  color: #e3893e;
+  display: right;
+  font-weight: 200px;
+  margin-right: 10px;
+  text-align: right;
+}
+
+.usersearch__container {
+  padding: 16px;
+  min-height: 10vh;
+  background: #000000;
+  background-size: cover;
+  display: flex;
+  flex-direction: column;
+  margin-left: auto;
```

Comment:
all those values are unnecessary

Reply:
Disagree. "display: flex" and "flex-direction: column" should stay in my opinion.

Reply:
The rest was removed.